modules/structs/repo_branch.go

```diff
@@ -22,7 +22,7 @@ type Branch struct {
 
 // BranchProtection represents a branch protection for a repository
 type BranchProtection struct {
-	BranchName                    string   `json:"branch_name"`
+	RuleName                      string   `json:"branch_name"`
```

Comment:
I think it's not a good idea to consider `branch_name` as `ruler_name`, it will be misunderstand